gateway-server/src/test/java/org/apache/knox/gateway/services/security/impl/ZookeeperRemoteAliasMonitorTest.java

```diff
@@ -234,13 +233,19 @@ public void testListener() throws Exception {
                 + ZookeeperRemoteAliasService.PATH_SEPARATOR + expectedAliasDev,
             zkAlias.encrypt(expectedPasswordDev).getBytes(StandardCharsets.UTF_8));
 
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      //
```

Comment:
Reset the interrupted thread status. Fixed a bunch of that here - https://issues.apache.org/jira/browse/KNOX-2130 couldn't figure out how to fail the build sadly :(

`Thread.currentThread().interrupt();`

Reply:
Well, this is test-only code, and no one of importance is checking the interrupted status further up the stack in this case. So, I'm not convinced there is much value in setting the interrupted status here.